tests/utils/test_databricks_utils.py

```diff
@@ -20,6 +20,7 @@ def test_no_throw():
     assert not databricks_utils.is_in_databricks_notebook()
     assert not databricks_utils.is_in_databricks_job()
     assert not databricks_utils.is_dbfs_fuse_available()
+    assert not databricks_utils.is_in_databricks_runtime()
```

Comment:
Can we add a test to make sure `is_in_databricks_runtime` returns True if `pyspark.databricks` can be imported?

```python
import sys
from unittest import mock


def test_is_in_databricks_runtime(tmpdir):

    fake_databricks_runtime = tmpdir.mkdir("pyspark")
    fake_databricks_runtime.join("__init__.py").write("")
    fake_databricks_runtime.join("databricks.py").write("")

    with mock.patch("sys.path", new=[tmpdir.strpath, *sys.path]):
        import pyspark.databricks  # pylint: disable=unused-import
```